### PR TITLE
legacy: Use KIND for e2e

### PR DESCRIPTION
**Pull request by rossf7**
Towards giantswarm/giantswarm#4606

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -53,7 +53,7 @@ required = [
   name = "github.com/giantswarm/certs"
 
 [[constraint]]
-  branch = "master"
+  branch = "update-guest"
```
Comment (rossf7) [inline Gopkg.toml]: TODO: Remove before merging.

File: Gopkg.toml [hunk 2]
```diff
@@ -65,7 +65,7 @@ required = [
   name = "github.com/giantswarm/e2etemplates"
 
 [[constraint]]
-  branch = "master"
+  branch = "update-guest"
```
Comment (rossf7) [inline Gopkg.toml]: TODO: Remove before merging.

**Comment by xh3b4sd** [comment]
Is this PR still relevant?

**Comment by rossf7** [comment]
> Is this PR still relevant?

Maybe its useful for https://github.com/giantswarm/giantswarm/issues/9519 but its also ancient.

I'll close it and add a note in the issue.

**Pull request closed without merging**